### PR TITLE
Add wrapper to set options dynamically

### PR DESCRIPTION
**Pull request by huachaohuang**

### CODE REVIEW

File: librocksdb_sys/crocksdb/crocksdb/c.h [hunk 1]
```diff
@@ -1651,6 +1655,24 @@ crocksdb_compact_files_cf(crocksdb_t*, crocksdb_column_family_handle_t*,
...
+
+extern C_ROCKSDB_LIBRARY_API void
+crocksdb_set_db_options(crocksdb_t* db,
+                        const char** names,
+                        const char** values,
+                        size_t num_options,
+                        char** errptr);
+
+extern C_ROCKSDB_LIBRARY_API void
+crocksdb_set_cf_options(crocksdb_t* db,
```
Comment (UncP) [inline librocksdb_sys/crocksdb/crocksdb/c.h]: Should we add `get_cf_options`?

Comment (huachaohuang) [reply librocksdb_sys/crocksdb/crocksdb/c.h]: We have, but it's called `get_options_cf`.

Comment (UncP) [reply librocksdb_sys/crocksdb/crocksdb/c.h]: Maybe change it to `get cf options` and put them together if it's ok.

Comment (huachaohuang) [reply librocksdb_sys/crocksdb/crocksdb/c.h]: We can't change the name for compatibility reason, but I have put them together.

**Comment by UncP** [comment]
LGTM

File: librocksdb_sys/crocksdb/crocksdb/c.h [hunk 2]
```diff
@@ -433,9 +433,27 @@ extern C_ROCKSDB_LIBRARY_API void crocksdb_disable_file_deletions(crocksdb_t* db
...
+                        const char** names,
+                        const char** values,
+                        size_t num_options,
+                        char** errptr);
+
 extern C_ROCKSDB_LIBRARY_API crocksdb_options_t* crocksdb_get_options_cf(
     const crocksdb_t* db, crocksdb_column_family_handle_t* column_family);
 
+extern C_ROCKSDB_LIBRARY_API void
+crocksdb_set_options_cf(crocksdb_t* db,
```
Comment (zhangjinpeng1987) [inline librocksdb_sys/crocksdb/crocksdb/c.h]: crocksdb_set_cf_options?

Comment (huachaohuang) [reply librocksdb_sys/crocksdb/crocksdb/c.h]: Well, we have a `get_options_cf` above, so I think `set_options_cf` makes it more consistent.

**Pull request merged**